### PR TITLE
Ansible BAT improvements

### PR DESCRIPTION
**Pull request by rbradford**
Some changes to make it easier to run BAT against an ansible built cluster.

### CODE REVIEW

**Comment by rbradford** [comment]
Need review from @albertomurillo and @markdryan

File: _DeploymentAndDistroPackaging/ansible/cleanup/ciao.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@
         - ./openrc
         - ./admin-ciaorc
         - ./demo-ciaorc
+        - ./bat-ciaorc
```
Comment (albertomurillo) [inline _DeploymentAndDistroPackaging/ansible/cleanup/ciao.yml]: This should be `../bat-ciaorc`
see https://github.com/01org/ciao/pull/1179

**Comment by rbradford** [comment]
@albertomurillo rebased and updated

**Comment by albertomurillo** [comment]
@rbradford one more change.
Add `{{ playbook_dir }}` to `clouds.yml` and `openrc` in `roles/keystone/templates`

Those files enable the ciao-cli to be used as well.

**Comment by rbradford** [comment]
@albertomurillo okay done.

**Comment by albertomurillo** [comment]
This can be merged now

**Comment by tpepper** [comment]
Pending @markdryan review

**Pull request merged**